storage/migrations/mysql/1_init.sql

```diff
@@ -87,158 +86,164 @@ CREATE TABLE IF NOT EXISTS `templates`
 ) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `campaigns` (
-  `id`            INTEGER UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  `user_id`       INTEGER UNSIGNED NOT NULL,
-  `name`          VARCHAR(191) NOT NULL,
-  `template_id`   INTEGER UNSIGNED,
-  `status`        VARCHAR(191) NOT NULL,
-  `created_at`    DATETIME(6) NOT NULL,
-  `updated_at`    DATETIME(6) NOT NULL,
-  `completed_at`  DATETIME(6) DEFAULT NULL,
-  `deleted_at`    DATETIME(6) DEFAULT NULL,
-  `started_at`    DATETIME(6) DEFAULT NULL,
-  FOREIGN KEY (`user_id`) REFERENCES users(`id`),
-  FOREIGN KEY (`template_id`) REFERENCES templates(`id`) ON DELETE SET NULL ON UPDATE CASCADE,
-  INDEX id_created_at (`id`, `created_at`)
+    `id`            INTEGER UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    `user_id`       INTEGER UNSIGNED NOT NULL,
+    `name`          VARCHAR(191) NOT NULL,
+    `template_id`   INTEGER UNSIGNED,
+    `status`        VARCHAR(191) NOT NULL,
+    `created_at`    DATETIME(6) NOT NULL,
+    `updated_at`    DATETIME(6) NOT NULL,
+    `completed_at`  DATETIME(6) DEFAULT NULL,
+    `deleted_at`    DATETIME(6) DEFAULT NULL,
+    `started_at`    DATETIME(6) DEFAULT NULL,
+    FOREIGN KEY (`user_id`) REFERENCES users(`id`),
+    FOREIGN KEY (`template_id`) REFERENCES templates(`id`) ON DELETE SET NULL ON UPDATE CASCADE,
+    INDEX idx_id_created_at (`id`, `created_at`),
+    INDEX idx_user_id (`user_id`)
 ) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `subscribers` (
-  `id`          BIGINT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
-  `user_id`     INTEGER UNSIGNED NOT NULL,
-  `name`        VARCHAR(191) DEFAULT NULL,
-  `email`       VARCHAR(191) NOT NULL,
-  `metadata`    JSON,
-  `blacklisted` TINYINT(1) NOT NULL,
-  `active`      TINYINT(1) NOT NULL,
-  `created_at`  DATETIME(6) NOT NULL,
-  `updated_at`  DATETIME(6) NOT NULL,
-  FOREIGN KEY (`user_id`) REFERENCES users(`id`),
-  INDEX id_created_at (`id`, `created_at`),
-  UNIQUE(`user_id`, `email`)
+    `id`          BIGINT UNSIGNED PRIMARY KEY AUTO_INCREMENT NOT NULL,
+    `user_id`     INTEGER UNSIGNED NOT NULL,
+    `name`        VARCHAR(191) DEFAULT NULL,
+    `email`       VARCHAR(191) NOT NULL,
+    `metadata`    JSON,
+    `blacklisted` TINYINT(1) NOT NULL,
+    `active`      TINYINT(1) NOT NULL,
+    `created_at`  DATETIME(6) NOT NULL,
+    `updated_at`  DATETIME(6) NOT NULL,
+    FOREIGN KEY (`user_id`) REFERENCES users(`id`),
+    INDEX idx_id_created_at (`id`, `created_at`),
+    INDEX idx_user_id (`user_id`),
+    INDEX idx_user_blacklist_active (`user_id`, `blacklisted`, `active`),
```

Comment:
Let's not add indexes prematurely here, we can remove these two.

Reply:
agreed